io.catenax.generic.digital_product_passport/2.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1142 @@
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Climate Change" .
+
+:dimensionValue a samm:Property ;
+   samm:preferredName "Dimension Value"@en ;
+   samm:description "The value associated with the unit."@en ;
+   samm:characteristic :DimensionValueCharacteristic ;
+   samm:exampleValue "20.5"^^xsd:float .
+
+:linearUnit a samm:Property ;
+   samm:preferredName "Linear Unit"@en ;
+   samm:description "The unit of a physical dimensions attribute. For linear attributes possible units are millimetre, metre and centimetre."@en ;
+   samm:characteristic :LinearUnitEnumeration ;
+   samm:exampleValue "unit:millimetre"^^samm:curie .
+
+:massUnit a samm:Property ;
+   samm:preferredName "Mass Unit"@en ;
+   samm:description "The unit of a physical dimensions attribute. For mass attributes weight possible units are gram and kilogram."@en ;
+   samm:characteristic :MassUnitEnumeration ;
+   samm:exampleValue "unit:kilogram" .
+
+:capacityUnit a samm:Property ;
+   samm:preferredName "Capacity Unit"@en ;
+   samm:description "The unit of a physical dimensions attribute. For capacity attributes the units are cubic metre, cubic centimetre, millilitre and litre."@en ;
+   samm:characteristic :CapacityUnitEnumeration ;
+   samm:exampleValue "unit:litre"^^samm:curie .
+
+:GtinTrait a samm-c:Trait ;
+   samm-c:baseCharacteristic :IdentifierCharacteristic ;
+   samm-c:constraint :GtinConstraint .
+
+:minConcentration a samm:Property ;
+   samm:preferredName "Minimum Concentration"@en ;
+   samm:description "The minimum concentration of the substance of concern at the level of the product. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Article 7:\n(5) (c) [...] concentration range of the substances of concern, at the level of the product [...]."@en ;
+   samm:characteristic :ConcentrationValue ;
+   samm:exampleValue "2.1"^^xsd:float .
+
+:maxConcentration a samm:Property ;
+   samm:preferredName "Maximum Concentration"@en ;
+   samm:description "The maximum concentration of the substance of concern at the level of the product. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Article 7:\n(5) (c) [...] concentration range of the substances of concern, at the level of the product [...]."@en ;
+   samm:characteristic :ConcentrationValue ;
+   samm:exampleValue "2.6"^^xsd:float .
+
+:concentrationSingle a samm:Property ;
+   samm:preferredName "Concentration Single"@en ;
+   samm:description "Concentration of the substance of concern at the level of the product. This can be either left: the maximum concentration, or  right: the concentration. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Article 7:\n(5) (c) the concentration, maximum concentration [...]."@en ;
+   samm:characteristic :EitherSingleConcentration ;
+   samm:exampleValue "2.41" .
```

Comment:
type?

Reply:
@bs-jokri see characteristic, or what do you mean? 
![image](https://github.com/eclipse-tractusx/sldt-semantic-models/assets/92307971/432841db-bfbb-4d8d-9426-d44c42b465df)

Reply:
invalid through the latest version